R/games.R

```diff
@@ -2178,11 +2178,8 @@ sample_fitness_pl <- static_power_law_game_impl
 #' @export
 #' @examples
 #'
-#' g <- sample_forestfire(10000, fw.prob = 0.37, bw.factor = 0.32 / 0.37)
-#' dd1 <- degree_distribution(g, mode = "in")
-#' dd2 <- degree_distribution(g, mode = "out")
-#' plot(seq(along.with = dd1) - 1, dd1, log = "xy")
-#' points(seq(along.with = dd2) - 1, dd2, col = 2, pch = 2)
+#' fire <- sample_forestfire(50, fw.prob = 0.37, bw.factor = 0.32 / 0.37)
+#' plot(fire)
```

Comment:
same example, more or less as in https://tidygraph.data-imaginist.com/reference/evolution_games.html#examples